src/estimator.cpp

```diff
@@ -139,7 +138,7 @@ void Estimator::run()
   state_.timestamp_us = now_us;
 
   // Crank up the gains for the first few seconds for quick convergence
-  if (now_us < (uint64_t)RF_.params_.get_param_int(PARAM_INIT_TIME)*1000)
+  if (now_us < static_cast<uint64_t>(RF_.params_.get_param_int(PARAM_INIT_TIME)*1000))
```

Comment:
I think it makes sense, from a clarity point of view, but it's not likely that the user will specify an initialization time of more than 20 minutes (the overflow of int32)